classes/argResolver.js

```diff
@@ -104,88 +79,51 @@ module.exports = class ArgResolver {
   }
 
   async string(arg, currentUsage, possible, repeat) {
-    if (currentUsage.possibles[possible].min && currentUsage.possibles[possible].max) {
-      if (arg.length < currentUsage.possibles[possible].min || arg.length > currentUsage.possibles[possible].max) {
-        if (currentUsage.type === "optional" && !repeat) {
-          return null;
-        } else if (currentUsage.possibles[possible].min === currentUsage.possibles[possible].max) {
-          throw `${currentUsage.possibles[possible].name} must be exactly ${currentUsage.possibles[possible].min} characters.`;
-        } else {
-          throw `${currentUsage.possibles[possible].name} must be between ${currentUsage.possibles[possible].min} and ${currentUsage.possibles[possible].max} characters.`;
-        }
-      } else {
-        return arg;
-      }
-    } else if (currentUsage.possibles[possible].min) {
-      if (arg.length < currentUsage.possibles[possible].min) {
-        if (currentUsage.type === "optional" && !repeat) {
-          return null;
-        }
-        throw `${currentUsage.possibles[possible].name} must be longer than ${currentUsage.possibles[possible].min} characters.`;
-      } else {
-        return arg;
-      }
-    } else if (currentUsage.possibles[possible].max) {
-      if (arg.length > currentUsage.possibles[possible].max) {
-        if (currentUsage.type === "optional" && !repeat) {
-          return null;
-        }
-        throw `${currentUsage.possibles[possible].name} must be shorter than ${currentUsage.possibles[possible].max} characters.`;
-      } else {
-        return arg;
-      }
-    } else {
-      return arg;
+    const min = currentUsage.possibles[possible].min;
+    const max = currentUsage.possibles[possible].max;
```

Comment:
You can use deconstruction here. `const { max } = currentUsage.possibles[possible];`